nat/upnp/discover.go

```diff
@@ -67,6 +69,12 @@ func discoverGateways() ([]GatewayDevice, error) {
 		return nil, err
 	}
 	defer client.Close()
+
+	// this IP address used to discover gateways - allow it
+	if _, err := firewall.AllowIPAccess("239.255.255.250"); err != nil {
```

Comment:
Maybe allow all multicast addresses `224.0.0.0/4`? UI discovery that we are using uses it too.

Reply:
UI discovery needs to do extra call itself where its needed